src/getTime.js

```diff
@@ -18,7 +18,11 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const gottenTimeArr = str.match(
```

Comment:
```suggestion
  const matches = str.match(
```
вынеси регулярное выражение в переменную чтобы не делать переносы (назови ее `pattern`)